pkg/model/components/etcdmanager/model.go

```diff
@@ -342,6 +343,13 @@ func (b *EtcdManagerBuilder) buildPod(etcdCluster kops.EtcdClusterSpec, instance
 		config.BackupInterval = fi.PtrTo(etcdCluster.Manager.BackupInterval.Duration.String())
 	}
 
+	if etcdCluster.Manager != nil && etcdCluster.Manager.BackupRetentionDays != nil {
```

Comment:
This doesn't appear to be defaulting to 90 days; it appears to default to not setting a retention. Or does etcd-manager do the 90-day default?

Reply:
I don't remember agreeing to change the behaviour for existing clusters. If I'm mistaking, I don't mind updating the PR.

Reply:
Let's discuss today, I think we should change the default if unspecified to match the 90 days we documented, but I think changing the default wouldn't go specifically here anyway (it should go where we populate the full cluster config, so it is more self-documenting).